libs/core/src/lib/theming/theming.service.ts

```diff
@@ -18,13 +18,15 @@ import { THEMING_CONFIG_TOKEN } from './tokens';
 
 @Injectable()
 export class ThemingService implements OnDestroy {
-    /** @hidden */
-    private readonly config = new BaseThemingConfig();
+    /**
+     * Theming configuration.
+     */
+    readonly config = new BaseThemingConfig();
 
     /**
-     * Current theme applied to the application.
+     * Observable of the current theme, applied to the application.
      */
-    get currentTheme(): Observable<CompleteThemeDefinition | null> {
+    get currentTheme$(): Observable<CompleteThemeDefinition | null> {
```

Comment:
Add another getter, with name `currentTheme` and log in that getter that this is deprecated, and return `currentTheme$`